api/locale/fi/LC_MESSAGES/django.po

```diff
@@ -5,78 +5,88 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-11-14 21:30+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
+"POT-Creation-Date: 2020-12-01 12:22+0200\n"
+"PO-Revision-Date: 2020-12-01 12:59+0200\n"
+"Language: fi\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Last-Translator: Sami Olmari <sami@olmari.fi>\n"
+"Language-Team: \n"
+"X-Generator: Poedit 2.4.2\n"
 
-#: api/models.py:18
+#: api/models.py:19
 msgid "User creation date"
 msgstr "Käyttäjän luontipäivä"
 
-#: api/models.py:19
+#: api/models.py:20
 msgid "Automatically set to now when user is create"
 msgstr "Automaattisesti asetettu täksi päiväksi kun käyttäjä on luotu"
 
-#: api/models.py:24
+#: api/models.py:25
 msgid "Last modified datetime"
 msgstr "Muokkauspäivä"
 
-#: api/models.py:25
+#: api/models.py:26
 msgid "Last time this user was modified"
 msgstr "Päiväys, koska viimeksi muokattu"
 
-#: api/models.py:29
+#: api/models.py:30
 msgid "Device name"
 msgstr "Laitteen nimi"
 
-#: api/models.py:30
+#: api/models.py:31
 msgid "Human readable name for the device, like door or laser"
 msgstr "Ihmisen luettava nimi laitteelle, kuten ovi tai laser"
 
-#: api/models.py:35
+#: api/models.py:36
 msgid "device id"
 msgstr "laitteen tunniste"
 
-#: api/models.py:36
+#: api/models.py:37
 msgid "used to know which device this was"
 msgstr "tunnistaa yksittäisen laitteen"
 
-#: api/models.py:49
+#: api/models.py:50
 msgid "Date of this entry"
 msgstr "Käytön päivämäärä"
 
-#: api/models.py:50
+#: api/models.py:51
 msgid "Automatically set to now when created"
 msgstr "Automaattisesti asetettu täksi päiväksi kun luotu"
 
-#: api/models.py:54
+#: api/models.py:55
 msgid "Granted"
 msgstr "Sallittu"
 
-#: api/models.py:54
+#: api/models.py:55
 msgid "Was the access granted"
 msgstr "Oliko pääsy sallittu"
 
-#: api/models.py:60
+#: api/models.py:61
 msgid "Device this event came from"
 msgstr "Laite, jolta tapahtuma tuli"
 
-#: api/models.py:67
+#: api/models.py:68
 msgid "NFC card id or phone number, or other field like that"
 msgstr "NFC-kortti, puhelinnumero, tai vastaava kenttä"
 
-#: api/models.py:74
+#: api/models.py:75
 msgid "NFC card"
 msgstr "NFC-kortti"
 
-#: api/models.py:75
+#: api/models.py:76
 msgid "NFC card this was mapped to, if any"
 msgstr "NFC-kortti johon tämä mäpättiin, jos sellainen oli"
+
+#: api/models.py:83
+msgid "User who claimed this"
+msgstr "Käyttäjä joka haltuunotti tämän"
```

Comment:
näissä on kyllä tuo englanninkielinenkin teksti aika huono :)

Reply:
Todennäköisesti ja jopa melko varmasti näin.